src/ichmod.cpp

```diff
@@ -193,6 +193,12 @@ usage() {
         " ",
         "The original owner of a dataObject can still set an ACCESS_LEVEL, even",
         "if their ACCESS_LEVEL has been set to 'null'.",
+        " ",
+        "Example Operations:",
+        " - irm - requires 'delete_object' or greater",
+        " - imv - requires 'delete_object' or greater, due to removal of old name",
+        " - iput - requires 'modify_object' or greater",
+        " - iget - requires 'read_object' or greater",
```

Comment:
open to better suggestions on how to do/show this type of information

Reply:
I understand why these are here, but they feel out of place.

It might be cleaner to document the permissions for an operation in that operation's help text. For example, `irm` grows a line or two explaining what permission is needed to delete an object. Same for `iput` and other icommands.

Reply:
I had written cool words here - but they got eaten.    Agreed on the weirdness/casing, was aiming for consistency with what we've had for a while.

Reply:
Oh, and I typed in the wrong box.    Yes, maybe these should be in different places... but since `ichmod` is where the permissions get changed... this also made some sense.   Maybe if the list were a little longer?  Some metadata levels too?   Or, not mentioning them related to the iCommands, but rather... just 'remove' and 'rename' and 'put' and 'get'?